lib/rules.js

```diff
@@ -117,9 +120,13 @@ Rules.prototype.check = function(value) {
       if (typeof(checks[property]) === "function") {
         var args = this.rules.properties[property];
 
-        if (!checks[property].apply(null, [value, args])) throw this.Error("%p failed %r with %v", property);
+        if (property === "in") {
+          if (!checks[property].apply(null, [value, args])) throw this.Error("'%p' failed '%r' with '%v' not one of [%args]", property, value, args);
```

Comment:
what about

```javascript
if (!checks[property].apply(null, [value, args])) {
  if (property === 'in')
     throw this.Error("'%p' failed '%r' with '%v' not one of [" + this.rules.properties.join(', ') + "]", property, value)
  else
     throw this.Error("'%p' failed '%r' with '%v'", property, value);
}
```

ths will keep `Error` simple and reduces the number of changes needed for this improvement